core-model/src/main/java/com/sequenceiq/cloudbreak/dto/StackDto.java

```diff
@@ -226,6 +226,12 @@ public Map<String, String> getParameters() {
         return stackParameters.stream().collect(Collectors.toMap(StackParameters::getKey, StackParameters::getValue));
     }
 
+    public List<InstanceMetadataView> getAllInstances() {
```

Comment:
As I checked it is all not terminated instance metadata, would you mind indicate the same in the method name?